### PR TITLE
Feat: Richtext field full screen mode, similar to markdown fieldtype

### PR DESCRIPTION
**Pull request by saravanan-inc**
https://github.com/rowyio/rowy/issues/752
Based on this issue, added a plugin based on the @tinymce/tinymce-react react library 
For more details on the plugin refer https://www.tiny.cloud/docs/plugins/opensource/fullscreen/

Added the fullscreen plugin in plugin array and toolbar 
image
image

### CODE REVIEW

**Comment by shamsmosowi** [comment]
Hi @saravanan-inc, are you able to run `yarn build` successfully? the perview deploy is currently failing

**Comment by saravanan-inc** [comment]
@shamsmosowi yeah yarn build is working in wsl2 using Ubuntu and also in windows 11
image

File: src/components/RichTextEditor.tsx [hunk 1]
```diff
@@ -121,6 +122,10 @@ const Styles = styled("div", {
         }
       : {},
   },
+  ".tox.tox-tinymce.tox-fullscreen": {
+    zIndex: 1203,
+    backgroundColor: "hsl(240, 9%, 7%)",
```
Comment (notsidney) [inline src/components/RichTextEditor.tsx:127]:
```suggestion
    "& .tox-tinymce.tox-fullscreen": {
      zIndex: theme.zIndex.modal,
      backgroundColor: theme.palette.background.paper,
    },
```
In the light theme, the background color is black and the text is black, making it unreadable. Please use MUI theme variables to ensure consistency throughout the app.

Once this change has been committed, this PR is good to merge.

**Review by notsidney — CHANGES_REQUESTED** [review]

File: src/components/RichTextEditor.tsx (hunk 1 above)
Comment (saravanan-inc) [reply src/components/RichTextEditor.tsx:127]: Ok

**Review by notsidney — APPROVED** [review]

**Comment by notsidney** [comment]
Thanks for pointing this out @saravanan-inc. The styles were correct, but they weren’t being applied because a more specific CSS selector from the 3rd-party component’s default styles was overriding our styles.

The side drawer’s z-index is `theme.zIndex.drawer - 1` (1199) and the full screen rich text editor is `theme.zIndex.modal` (1300). [(MUI z-index values)](https://mui.com/material-ui/customization/z-index) The transparent background was making the side drawer visible, but you could not interact with the side drawer.

**Pull request merged**